From affca46a95820ae383b701d5af6ab98b73b3a8a7 Mon Sep 17 00:00:00 2001
From: Ray Chang <ray.chang@technexion.com>
Date: Fri, 14 Nov 2025 13:57:27 +0800
Subject: [PATCH] wpa_supplicant: defconfig: enable dbus control interface for
 connman

Connman requires the DBus control interface to be enabled in order
to control wireless network interfaces.

Upstream-Status: Pending
---
 wpa_supplicant/defconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/wpa_supplicant/defconfig b/wpa_supplicant/defconfig
index ec7db31..350d693 100644
--- a/wpa_supplicant/defconfig
+++ b/wpa_supplicant/defconfig
@@ -379,10 +379,10 @@ CONFIG_INTERNAL_LIBTOMMATH=y
 
 # Add support for new DBus control interface
 # (fi.w1.wpa_supplicant1)
-#CONFIG_CTRL_IFACE_DBUS_NEW=y
+CONFIG_CTRL_IFACE_DBUS_NEW=y
 
 # Add introspection support for new DBus control interface
-#CONFIG_CTRL_IFACE_DBUS_INTRO=y
+CONFIG_CTRL_IFACE_DBUS_INTRO=y
 
 # Add support for loading EAP methods dynamically as shared libraries.
 # When this option is enabled, each EAP method can be either included
